modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2111,7 +2111,7 @@ private void _insertIntoTable(
 			for (ObjectField objectField : objectFields) {
 				Object value = values.get(objectField.getName());
```

Comment:
Hi @jcampoy, can we move [the line](https://github.com/liferay-objects/liferay-portal/pull/964/commits/9b8b08f74a24b599820ce715afc40d2249491ba5#diff-520634a879bf903adc87e7625aa3ffd7470975dac938d58c51c40d478b352dabR2112) to before line 2121 like the code below?

```
	for (ObjectField objectField : objectFields) {
		if (!values.containsKey(objectField.getName())) {
			continue;
		}

		Column<?, ?> column = dynamicObjectDefinitionTable.getColumn(
			objectField.getDBColumnName());

		Object value = values.get(objectField.getName());

		_setColumn(
			preparedStatement, index++, column.getSQLType(), value);
	}
```

Reply:
Hi @seltong , see my changes [a19f7b4](https://github.com/liferay-objects/liferay-portal/pull/964/commits/a19f7b46aea99990bdf36026e24a595adf50407c), if you think it is better to continue saving the value in a new Object, please, go ahead and change it and resend, we would like to fix it asap.